src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    private TaskList tasks;
+    private Storage storage;
+    private Ui ui;
+    private Parser parser;
+
+    public Duke() {
+        this.tasks = new TaskList();
+        this.storage = new Storage("C:/Users/Jeremias/Documents/GitHub/ip/data/", "data.txt");
+        this.ui = new Ui();
+        this.parser = new Parser();
+    }
+
+    public static void main(String[] args) {
+        Duke duke = new Duke();
+        duke.run();
+    }
+
+    /**
+     * Starts the Duke program.
+     */
+    private void run() {
+        ui.printGreeting();
+        storage.loadTaskList(tasks);
+        Scanner sc = new Scanner(System.in);
+        boolean exit = false;
```

Comment:
Perhaps this boolean variable could be better named to sound like a boolean. E.g. `shouldExit`